.evergreen/config.yml

```diff
@@ -431,6 +432,7 @@ functions:
           export MONGO_ATLAS_TLS11_URI_SRV='${MONGO_ATLAS_TLS11_URI_SRV}'
           export MONGO_ATLAS_TLS12_URI='${MONGO_ATLAS_TLS12_URI}'
           export MONGO_ATLAS_TLS12_URI_SRV='${MONGO_ATLAS_TLS12_URI_SRV}'
+          export PROJECT_DIRECTORY='${PROJECT_DIRECTORY}'
```

Comment:
this is needed because configure-rust.sh uses it. for all of the other tasks we run $PREPARE_SHELL which already does this for us.